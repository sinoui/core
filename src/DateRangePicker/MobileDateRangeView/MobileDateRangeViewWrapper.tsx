import styled from 'styled-components';
import Paper from '@sinoui/core/Paper';

const disabledProps = ['in'];
const MobileDateRangeViewWrapper = styled(Paper).withConfig({
  shouldForwardProp: (prop) => !disabledProps.includes(prop),
})`
  width: 328px;

  .sinoui-week-title-bar {
    padding: 0 12px;
  }
`;

export default MobileDateRangeViewWrapper;
