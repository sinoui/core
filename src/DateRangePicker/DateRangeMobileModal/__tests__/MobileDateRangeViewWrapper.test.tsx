import React from 'react';
import { ThemeProvider } from 'styled-components';
import { defaultTheme } from '@sinoui/theme';
import renderer from 'react-test-renderer';
import MobileDateRangeViewWrapper from '../MobileDateRangeViewWrapper';

it('快照测试--> 渲染', () => {
  const tree = renderer
    .create(
      <ThemeProvider theme={defaultTheme}>
        <MobileDateRangeViewWrapper>123</MobileDateRangeViewWrapper>
      </ThemeProvider>,
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
