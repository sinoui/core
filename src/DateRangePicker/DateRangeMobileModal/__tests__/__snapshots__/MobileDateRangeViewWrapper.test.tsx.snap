// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`快照测试--> 渲染 1`] = `
<div
  className="sc-AxjAm sc-AxirZ esVjVJ sinoui-paper"
>
  123
</div>
`;
